### ISSUE TITLE
Issue leaving privileged mode on "hp" personality

### ISSUE THREAD

**Issue by JeroenvIS** [ollyg/Net-CLI-Interact#38]
Using Net::Appliance::Session (which in turn uses Net::CLI::Interact), I get errors when interacting with HP ProCurve / Aruba AOS-S devices (using the "hp" personality). It appears that leaving the privileged mode fails.

A simplified part of the script that I used for reproduction:
```
     my $s = Net::Appliance::Session->new({
        personality => 'hp',
        transport => 'SSH',
        host => 'test-hp-5406.civ.utwente.nl',
    });
   try {
        $s->connect({ username => 'manager', password => $pass });
        $s->begin_privileged;
        $s->begin_configure;
        $s->cmd('hostname test-hp-5406', {timeout => 20});
        $s->end_configure;
        $s->end_privileged;
    }
    catch {
        warn "failed to execute command: $_";
    }
    finally {
        $s->close;
    };
```

results in the output below, with log level at notice:

```
[   0.004091] pr finding prompt
[   0.016949] tr creating Net::Telnet wrapper for ssh
[   0.017007] tr connecting with:  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o CheckHostIP=no -l manager test-hp-5406.civ.utwente.nl
[  11.466179] pr failed: [read timed-out at /usr/share/perl5/Net/CLI/Interact/Transport/Wrapper/Net_Telnet.pm line 33

], sending WAKE_UP and trying again
[  11.466654] pr finding prompt
[  11.484138] pr output matching prompt was "test-hp-5406# "
[  11.488060] en running macro disable_paging
[  11.488467] en executing actions
[  11.489114] di queueing data for send: "no page"
[  11.536260] pr output matching prompt was "no pagetest-hp-5406# "
[  11.536841] en running macro begin_configure
[  11.537074] en executing actions
[  11.537572] di queueing data for send: "configure terminal"
[  11.584083] pr output matching prompt was "configure terminaltest-hp-5406(config)# "
[  11.584549] en running command hostname test-hp-5406
[  11.584734] en executing actions
[  11.585262] di queueing data for send: "hostname test-hp-5406"
[  11.691052] pr output matching prompt was "hostname test-hp-5406test-hp-5406(config)# "
[  11.691469] en running macro end_configure
[  11.691720] en executing actions
[  11.692305] di queueing data for send: "exit"
[  11.744223] pr output matching prompt was "exittest-hp-5406# "
[  11.744758] en running macro end_privileged
[  11.745006] en executing actions
[  11.745562] di queueing data for send: "disable"
[  11.792188] pr output matching prompt was "test-hp-5406# "
failed to execute command: should have left privileged mode but prompt still matches at /usr/share/perl5/Net/Appliance/Session/Engine.pm line 125.
[  11.792815] en running macro end_privileged
[  11.793102] en executing actions
[  11.793674] di queueing data for send: "disable"
[  11.840262] pr output matching prompt was "test-hp-5406# "
Execution of finally() block CODE(0x55ccba41c858) resulted in an exception, which *CAN NOT BE PROPAGATED* due to fundamental limitations of Perl. Your program will continue as if this event never took place. Original exception text follows:

should have left privileged mode but prompt still matches at /usr/share/perl5/Net/Appliance/Session/Engine.pm line 125.
```

**Comment by JeroenvIS** [comment]
After further investigation, I locally edited the "hp" phrasebook. It inherits from "cisco" where 'disable' is used to leave privileged mode. HP ProCurve doesn't implement this command. It uses 'exit' for various context changes, but also has the specific 'end' command to leave configure mode and 'logout' to log out.

The changes in PR #37 work well for me: no more errors upon leaving privileged mode and no ambiguous use of 'exit', which reduces the chance of executing a context change that it different from the intended one. Please consider merging this in a new release.

**Comment by ollyg** [comment]
Thanks @JeroenvIS I have merged the PR #37 so fixed in 2906542
